ovn-tester/ovn_tester.py

```diff
@@ -312,6 +351,17 @@ def run_base_cluster_bringup(ovn, bringup_cfg, global_cfg):
         ovn.provision_lb_group()
 
 
+def start_process_monitor(container):
+    log.info(f'Starting process monitor for {container}')
+    cmd = shlex.split(
+        f'bash -c "nohup python3 /tmp/process-monitor.py '
+        f'-s {container} '
+        f'-o /var/log/process-stats.json '
+        f'-x /tmp/process-monitor.exit"'
+    )
+    subprocess.Popen(cmd)
```

Comment:
Can we have a TesterNode class instead and not duplicate this function?
Another option is to start the process monitor from ansible, i.e. the place where we actually start that container.

Reply:
It's not exactly a duplicate of the other function, since this is not using ssh or `docker exec` like the other nodes and instead runs a process directly on the running container.

However, the suggestion to run it from ansible sounds good too.

Reply:
I guess, SSHing to itself should not be problem.  But ansible way is also fine.